src/content/docs/apis/nerdgraph/examples/nerdgraph-manage-users.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: "NerdGraph tutorial: Manage users"
+tags:
+  - APIs
+  - NerdGraph
+metaDescription: "How to use New Relic's NerdGraph API to manage users and return user information."
+---
+
+You can use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage users. You can edit their user type and email address, and return other types of user information. 
+
+For how to do this in the UI, see the [user management UI docs](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks). 
+
+## Requirements [#requirements]
+
+Some requirements for managing users via NerdGraph: 
+* You must be a user on [our newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models#determine-user-model). Other permissions-related requirements: 
```

Comment:
It's a minor thing, but I generally like to avoid time-based descriptions of features. Technically it may be our "newer" user model for a long time, but it may not always be. I'd rather just use the actual user model name here.

Reply:
we actually call it officially just 'the newer user model' when necessary. otherwise we avoid talking about it. this was change from calling it the 'New Relic One user model'. should be going away soon entirely, on plus side